app/models/profile.rb

```diff
@@ -70,6 +80,14 @@ def deactivate(reason)
     update!(active: false, deactivation_reason: reason)
   end
 
+  def deactivation_checks
```

Comment:
1. Can we name this with a `?` because it's a boolean?
2. how about something like `has_any_deactivation_reason?`